substrate/client/chain-spec/src/lib.rs

```diff
@@ -76,12 +281,9 @@
 //! /// A chain spec supporting forkable `Extension`.
 //! pub type MyChainSpec2<G> = GenericChainSpec<G, Forks<BlockNumber, Extension>>;
 //! ```
-//!
-//! It's also possible to have a set of parameters that is allowed to change
-//! with block numbers (i.e. is forkable), and another set that is not subject to changes.
-//! This is also possible by declaring an extension that contains `Forks` within it.
-//!
-//!
+//! It's also possible to have a set of parameters that is allowed to change with block numbers
+//! (i.e. is forkable), and another set that is not subject to changes. This is also possible by
+//! declaring an extension that contains `Forks` within it.
```

Comment:
```suggestion
//! It's also possible to have a set of parameters that are allowed to change with block numbers
//! (i.e., they are forkable), and another set that is not subject to changes. This can also be
//! achieved by declaring an extension that contains [`Forks`] within it.
```